.github/scripts/jpackage.sh

```diff
@@ -19,4 +19,4 @@ $JPACKAGE_HOME/bin/jpackage \
 --verbose \
 --runtime-image app/target/runtime \
 --dest $INSTALL_DIR \
-"$@"
\ No newline at end of file
+"$@"
```

Comment:
For some reason the GH editor added new line at the end, but this should not be an issue.